monai/losses/dice.py

```diff
@@ -823,8 +823,10 @@ def __init__(
         self,
         include_background: bool = True,
         to_onehot_y: bool = False,
-        sigmoid: bool = False,
-        softmax: bool = False,
+        sigmoid_dice: bool = False,
```

Comment:
Hi @j-morano, thanks for the contributions!
We would suggest using `deprecated_arg` to add such changes.
https://github.com/Project-MONAI/MONAI/blob/b92b2ce8aa25e53c8e1824da0a063b9e5239f1e4/monai/utils/deprecate_utils.py#L123

You can find an example here:
https://github.com/Project-MONAI/MONAI/blob/b92b2ce8aa25e53c8e1824da0a063b9e5239f1e4/monai/metrics/meandice.py#L251